src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("Forecast Details", () => {
+  const validProps = {
+    date: 1525046400000,
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    humidity: 30,
+    wind: {
+      speed: 60,
+      direction: "n",
+    },
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(<ForecastDetails forecast={validProps} />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders correct values for props", () => {
+    const { getByText } = render(<ForecastDetails forecast={validProps} />);
```

Comment:
you should use `screen` to access these kind of methods, such as `getByText`, instead of destructuring this out of the render. As you've done in the `renders Weathere App header` in App.test.js. The only one you should destructure out of the render is `asFragment`